packages/core/database/src/metadata/index.ts

```diff
@@ -38,7 +38,9 @@ export const createMetadata = (models: Model[] = []): Metadata => {
   for (const model of _.cloneDeep(models)) {
     metadata.add({
       ...model,
-      attributes: model.attributes,
+      attributes: {
+        ...model.attributes,
+      },
```

Comment:
I believe we had an issue about attributes in the schema being mutated before, glad about this change! 🕺

Reply:
in theory it's deep cloned earlier but I'll need it in the future :)